t/form-get.t

```diff
@@ -0,0 +1,20 @@
+use Cro::WebApp::Form;
+use Cro::WebApp::Template::Repository;
+use Test;
+
+class Search does Cro::WebApp::Form { has $.query is search }
+class Upload does Cro::WebApp::Form { has $.file  is file   }
+
+constant template = parse-template ｢<&form($_, :method('get'))>｣;
+
+with template.render: Search.empty {
+    like $_, / 'method="get"' /, 'Form method is GET';
+
+    unlike $_, / '__CSRF_TOKEN' /, 'CSRF token is omitted';
+}
+
+throws-like { template.render: Upload.empty }, Exception,
+    :message("Can't have type 'file' in a GET form"),
```

Comment:
The test would be more robust with a typed exception too

Reply:
done